.gitignore

```diff
@@ -25,7 +25,7 @@ cachegrind.out*
 # Codelite (among others) project files
 *.project
 # ctags index files
-tags
+*tag*
```

Comment:
I put it in there because my IDE puts a .tags and .tag1 in the base folder.  I put the ignore in there to keep from adding the ctags files.  I did it that way so it would catch the tags and my .tags